src/bin/playground.cpp

```diff
@@ -154,37 +158,29 @@ void write_dict_segment_to_disk(const std::shared_ptr<DictionarySegment<int>> se
    * As a next step we should use the AttributeVectorCompressionID to define the type of the FixedWidthIntegerVector
    * and perhaps also write out the type of the DictionarySegment.
    */
-  std::ofstream chunk_file;
-  chunk_file.open(filename, std::ios::out | std::ios::binary | std::ios::app);
 
   // We will later mmap to an uint32_t vector/array. Therefore, we store all metadata points as uint32_t.
   // This wastes up to three bytes of compression per metadata point but makes mapping much easier.
-  export_value(chunk_file, static_cast<uint32_t>(segment->dictionary()->size()));
-  export_value(chunk_file, static_cast<uint32_t>(segment->attribute_vector()->size()));
-
-  // std::cout << "DictionarySize: " << segment->dictionary()->size() << " AttributeVectorSize: " << segment->attribute_vector()->size() << std::endl;
+  export_value(FILESTREAM, static_cast<uint32_t>(segment->dictionary()->size()));
+  export_value(FILESTREAM, static_cast<uint32_t>(segment->attribute_vector()->size()));
 
   const auto compressed_vector_type_id = static_cast<uint32_t>(infer_compressed_vector_type_id<int>(*segment));
-  export_value(chunk_file, compressed_vector_type_id);
+  export_value(FILESTREAM, compressed_vector_type_id);
 
-  export_values<int32_t>(chunk_file, *segment->dictionary());
+  export_values<int32_t>(FILESTREAM, *segment->dictionary());
 
-  export_compressed_vector(chunk_file, *segment->compressed_vector_type(),
+  export_compressed_vector(FILESTREAM, *segment->compressed_vector_type(),
                           *segment->attribute_vector());
-  chunk_file.close();
 }
 
 std::vector<uint32_t> generate_segment_offset_ends(const std::shared_ptr<Chunk> chunk) {
   const auto segment_count = chunk->column_count();
   auto segment_offset_ends = std::vector<uint32_t>(segment_count);
 
-
+  auto offset_end = static_cast<uint32_t>(4 + 4 * segment_count);
```

Comment:
why first 4 ? Refactor into variable name
refactor second 4 to byte_size_multiplyier

Reply:
Done.